### PR TITLE
Add StringifyPublicKey method for NodeInfo

### PR DESCRIPTION
**Pull request by acid-ant**
Closes https://github.com/nspcc-dev/neofs-node/issues/1765

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: crypto/util.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package neofscrypto
+
+import "encoding/hex"
+
+// StringifyKeyBinary returns string with HEX representation of source
```
Comment (cthulhu-rider) [inline crypto/util.go:5]: I suggest to also mention that format can be changed and it's unsafe to rely on it beyond human-readable output.

File: netmap/node_info.go [hunk 2]
```diff
@@ -178,6 +179,11 @@ func (x NodeInfo) PublicKey() []byte {
 	return x.m.GetPublicKey()
 }
 
+// StringifyPublicKey returns HEX representation of PublicKey
+func (x NodeInfo) StringifyPublicKey() string {
```
Comment (cthulhu-rider) [inline netmap/node_info.go:183]: Can we make it a function over `NodeInfo`? It provides access to public key via method.

File: crypto/util.go (hunk 1 above)
Comment (acid-ant) [reply crypto/util.go]: Updated.

File: netmap/node_info.go (hunk 2 above)
Comment (acid-ant) [reply netmap/node_info.go:183]: Not sure that I understand your point, please check new changes.

Comment (cthulhu-rider) [reply netmap/node_info.go:183]: I mean now you can declare it as func `StringifyNodeKey(node NodeInfo) string` since no private data is used.

Comment (acid-ant) [reply netmap/node_info.go]: Got it, fixed.

**Review by cthulhu-rider — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Pull request merged**